com.unity.shadergraph/Editor/Drawing/PreviewManager.cs

```diff
@@ -486,10 +488,29 @@ public void RenderPreviews(bool requestShaders = true)
                 CollectPreviewProperties(m_NodesPropertyChanged, perMaterialPreviewProperties);
                 m_NodesPropertyChanged.Clear();
 
-                // timed nodes change every frame, so must be drawn
+                // timed nodes are animated, so they should be updated regularly
                 // (m_TimedPreviews has been pre-propagated downstream)
                 // HOWEVER they do not need to collect properties. (the only property changing is time..)
-                m_PreviewsToDraw.UnionWith(m_TimedPreviews);
+
+                // get current window FPS, clamp to what we consider a valid range
+                // this is probably not accurate for multi-monitor.. but should be relevant to at least one of the monitors
```

Comment:
Not sure how this behaves for variable refresh card/monitor pairs either.

Reply:
Yeah, partially why I put the clamp in there -- just in case you get weird numbers somewhere, it will clamp it to a decent range.